packages/microeditor-envelope/src/__tests__/AppFormerGwtApi.test.ts

```diff
@@ -0,0 +1,33 @@
+import { AppFormerGwtApi } from '../gwt/AppFormerGwtApi';
+import { DummyEditor } from './DummyEditor';
+import * as AppFormer from "appformer-js-core";
+import { GwtEditor } from '../gwt/GwtEditor';
+
+class DummyGWTEditor {
```

Comment:
We should go for camel-case acronyms in class names too, so this class should be `DummyGwtEditor`.

Reply:
Also on the variables